crates/forge/src/runner.rs

```diff
@@ -310,6 +309,30 @@ impl<'a> ContractRunner<'a> {
         }
 
         let has_invariants = self.contract.abi.functions().any(|func| func.is_invariant_test());
```

Comment:
this call looks out of place here,
can we move this closer to where we check this?

Reply:
yep, grouped it with tmp_tracing in https://github.com/foundry-rs/foundry/pull/8106/commits/21a790b52180ddc0d96a8c3cf89a880f5c8ca50e